source/grid/grid_in.cc

```diff
@@ -433,35 +434,41 @@ GridIn<dim, spacedim>::read_vtk(std::istream &in)
       for (unsigned int i = 0; i < n_ints; ++i)
         in >> tmp_int;
 
+
+      // Processing the CELL_DATA and FIELD_DATA sections
```

Comment:
Are `CELL_DATA` and `FIELD_DATA` the same?

Reply:
The CELL_DATA section is used to specify Material or Manifold IDs at the cells and only allows int type. The new FIELD_DATA section is to specify other properties at the cells, (such as pore radius, density, etc.) and allows double datatypes.